### PR TITLE
bibox withdrawal fees and refactor request

### PR DESCRIPTION
**Pull request by frosty00**
If a currency is not in fetch_balance instead of ignoring it and leaves the result {} - so that a KeyError is raised in python when you do `exchange.fetch_balance()['currency']`, fill those values with 0 for total, used and free.

The `fetch_funding_fees` is simillar to binance in that it makes many requests by default but this is to conform to the standard of loading all funding fees.

### CODE REVIEW

**Comment by frosty00** [comment]
I would like to remove 
```
if (body.length > 0) {
    if (body[0] === '{') {
        let response = JSON.parse (body);
```
 from all exchange subclasses in favour for a unified approach. Could you take a look at #2296 as I spend quite alot of time writing it and have been using in Python since I made the PR without any problems :)

**Comment by kroitor** [comment]
For some reason the added endpoint returns [ExchangeError] bibox {"error":{"code":"3000","msg":"请求参数错误"},"cmd":"transfer/transferOutInfo"}

**Comment by kroitor** [comment]
Looks like it's being sent with an empty `"body"` in the `cmd`... Debugging it now.

**Comment by kroitor** [comment]
Sorry, that was an error in my test, please ignore it. Your code is fine. Merging! Thanks! )

**Comment by kroitor** [comment]
Will merge #2296 as well after #2397 ) Hope to get it done very soon.

File: js/bibox.js [hunk 1]
```diff
@@ -557,47 +596,30 @@ module.exports = class bibox extends Exchange {
...
+                        }
+                    }
+                    throw new ExchangeError (this.id + ': "error" in response: ' + body);
+                }
+                if (!('result' in response))
+                    throw new ExchangeError (this.id + ' ' + body);
+                if (method === 'GET') {
+                    return response;
+                } else {
+                    return response['result'][0];
```
Comment (kroitor) [inline js/bibox.js]: Oh, now I see why it's broken... It never uses this return...

Comment (kroitor) [reply js/bibox.js]: No worries, will fix it shortly.

Comment (frosty00) [reply js/bibox.js]: Working in python now cheers

Comment (kroitor) [reply js/bibox.js]: Thanks, @frosty00 !)

**Pull request merged**